<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PedalVinylStopFrame</class>
 <widget class="QWidget" name="PedalVinylStopFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToasterStompEditLCD" name="lcdDisplay">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>75</y>
     <width>420</width>
     <height>115</height>
    </rect>
   </property>
   <property name="color">
    <enum>QToasterLCD::White</enum>
   </property>
   <property name="currentPage">
    <enum>QToasterLCD::Page1</enum>
   </property>
   <property name="maxPage">
    <enum>QToasterLCD::Page1</enum>
   </property>
   <property name="stompName">
    <string>Pure Booster</string>
   </property>
   <property name="value1Title">
    <string>Mix</string>
   </property>
   <property name="value2Title">
    <string/>
   </property>
   <property name="value3Title">
    <string/>
   </property>
   <property name="value4Title">
    <string>Volume</string>
   </property>
   <property name="value5Title">
    <string>WPedal2Pitch</string>
   </property>
   <property name="value6Title">
    <string/>
   </property>
   <property name="value7Title">
    <string/>
   </property>
   <property name="value8Title">
    <string/>
   </property>
   <property name="value9Title">
    <string/>
   </property>
   <property name="value10Title">
    <string/>
   </property>
   <property name="value11Title">
    <string/>
   </property>
   <property name="value12Title">
    <string/>
   </property>
   <property name="value13Title">
    <string/>
   </property>
   <property name="value14Title">
    <string/>
   </property>
   <property name="value15Title">
    <string/>
   </property>
   <property name="value16Title">
    <string/>
   </property>
   <property name="scaleView">
    <enum>QToasterStompEditLCD::Empty</enum>
   </property>
  </widget>
  <widget class="QToasterDial" name="mixDial">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::Uni</enum>
   </property>
   <property name="minValue">
    <double>0.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>100.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>1.000000000000000</double>
   </property>
   <property name="precision">
    <UInt>0</UInt>
   </property>
   <property name="unit">
    <string>%</string>
   </property>
   <property name="isActive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToasterDial" name="volumeDial">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::Bi</enum>
   </property>
   <property name="minValue">
    <double>-5.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>5.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="unit">
    <string/>
   </property>
  </widget>
  <widget class="QToasterDial" name="dummy3Dial">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::Uni</enum>
   </property>
   <property name="minValue">
    <double>0.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>1.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>1.000000000000000</double>
   </property>
   <property name="unit">
    <string/>
   </property>
   <property name="isActive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToasterDial" name="dummy2Dial">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::Uni</enum>
   </property>
   <property name="minValue">
    <double>0.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>1.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="isActive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToasterDial" name="dummy8Dial">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="isActive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToasterDial" name="dummy6Dial">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="isActive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToasterDial" name="dummy7Dial">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="isActive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToasterEnumDial" name="wahPedalToPitchDial">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterEnumDial::Small</enum>
   </property>
   <property name="values">
    <stringlist>
     <string>Off</string>
     <string>On</string>
    </stringlist>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QToasterDial</class>
   <extends>QWidget</extends>
   <header>qtoasterdial.h</header>
  </customwidget>
  <customwidget>
   <class>QToasterEnumDial</class>
   <extends>QWidget</extends>
   <header>qtoasterenumdial.h</header>
  </customwidget>
  <customwidget>
   <class>QToasterStompEditLCD</class>
   <extends>QWidget</extends>
   <header>QToasterStompEditLCD.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>volumeDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue4(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue1(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wahPedalToPitchDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue5(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
